utils/cloudWatchEvent.js

```diff
@@ -0,0 +1,68 @@
+const {
+    PutRuleCommand,
+    PutTargetsCommand,
+} = require('@aws-sdk/client-eventbridge');
+
+const { addPermissions } = require('./addPermissions');
+const { CLOUDWATCH_EVENT } = require('./constants');
+
+const { EventBridgeClient } = require('@aws-sdk/client-eventbridge');
+
+exports.cloudWatchEvent = async (
+    name,
+    rangeTime,
+    region,
+    accessKey,
+    secretKey,
+    iamRoleArnEvent,
+    accountId,
+) => {
+    try {
+        const cweClient = new EventBridgeClient({
+            region: region,
+            credentials: {
+                accessKeyId: accessKey,
+                secretAccessKey: secretKey,
+            },
+        });
+        const paramsTarget = {
+            Rule: CLOUDWATCH_EVENT,
+            Targets: [
+                {
+                    Arn: `arn:aws:lambda:${region}:${accountId}:function:${name}`, //LAMBDA_FUNCTION_ARN
+                    Id: 'myCloudWatchEventsTarget',
+                },
+            ],
+        };
+        const paramsRule = {
+            Name: CLOUDWATCH_EVENT,
+            RoleArn: iamRoleArnEvent,
```

Comment:
Suggestion: to improve user experience can we create the `iamRoleArnEvent` using AWS SDK, instead of making the user manually create it and copy-paste the arn?